src/pages/iou/SplitExpensePage.tsx

```diff
@@ -267,6 +323,9 @@ function SplitExpensePage({route}: SplitExpensePageProps) {
                 transactionID: item?.transactionID ?? CONST.IOU.OPTIMISTIC_TRANSACTION_ID,
                 currencySymbol,
                 onSplitExpenseAmountChange,
+                mode: isPercentageMode ? CONST.IOU.SPLIT_TYPE.PERCENTAGE : CONST.IOU.SPLIT_TYPE.AMOUNT,
+                percentage,
+                onSplitExpensePercentageChange,
```

Comment:
Can we combine this method with `onSplitExpenseAmountChange` and rename it to something like `onSplitExpenseValueChange`.